packages/react/src/components/Accordion/__tests__/accordion-avt.md

```diff
@@ -0,0 +1,34 @@
+# Accordion Component Accessibility Verification Testing
```

Comment:
Would it make sense to have this at `components/Accordion/AVT.md`?

Reply:
matched the format of the Link md. Easier for ctrl-p searchin'.